docs/design/cincinnati.md

```diff
@@ -216,31 +216,32 @@ Determining valid graph traversals can be tricky to do by hand. The following [`
 
 #### Example ####
 
-The following will fetch the graph from a remote endpoint and determine the set of versions to which "1.0.0" can transition:
+The following will fetch the graph from the remote endpoint and determine the set of versions to which OpenShift version 4.2.13 can transition in stable-4.2 channel:
+
+```json
```

Comment:
nit: the `$ curl ...` means this should be a:

    ```console

block.  If you want pretty JSON, you could say:

    ```console
    $ curl --silent --header 'Accept:application/json' 'https://api.openshift.com/api/upgrades_info/v1/graph?arch=amd64&channel=stable-4.2' | jq '. as $graph | $graph.nodes | map(.version == "4.2.13") | index(true) as $orig | $graph.edges | map(select(.[0] == $orig)[1]) | map($graph.nodes[.])'
    ```

    which will produce output like:

    ```json
    [
      {
    ...
    ```

or some such.

Reply:
Fixed in 8935d2b.